packages/components/src/components/filter-dropdown/filter-dropdown.tsx

```diff
@@ -1,25 +1,33 @@
 import * as React from 'react';
-import PropTypes from 'prop-types';
 import { isMobile } from '@deriv/shared';
-import Dropdown from '../dropdown/dropdown.jsx';
-import SelectNative from '../select-native/select-native.jsx';
+import Dropdown from '../dropdown/index.js';
+import SelectNative from '../select-native/index.js';
+
+type FilterDropdownProps = {
+    dropdown_className: string;
+    dropdown_display_className: string;
+    filter_list: Array<any>;
```

Comment:
Can we add any other type instead of any here?

Reply:
```suggestion
    filter_list: Array<{
    text: string;
    value: string;
}>;
```